blogposts/code-intelligence-on-sourcegraph-com.md

```diff
@@ -0,0 +1,17 @@
+---
+title: 'Code intelligence on sourcegraph.com'
+author: 'Michael Fromberger'
+publishDate: 2020-04-14T00:00-07:00
+tags: [
+  "blog"
+]
+slug: code-intelligence-on-sourcegraph-com
+heroImage: TBD
+published: false
+---
+
+At Sourcegraph we are working toward a long term goal of making code intelligence results both fast and precise. Today, Sourcegraph provides basic code intelligence for many languages [using our search engine](https://docs.sourcegraph.com/user/code_intelligence/basic_code_intelligence). It's fast, requires zero configuration, and covers many use cases, but the results are not guaranteed to be completely accurate.  To get more precise answers we also use language-specific indexers to generate data in Language Server Index Format (LSIF), for important navigation queries like Go-to-Definition and Find References. Indexers already exist for several languages, including C++, Go, TypeScript, and Dart, and more are in-progress (visit https://lsif.dev for a list of the indexers we've heard of so far).
```

Comment:
>visit https://lsif.dev for a list of the indexers we've heard of so far

This makes it sound like we are the only ones maintaining/updating lsif.dev. What do you think about something like:

>visit https://lsif.dev for a list of indexers